02-Lab/ImageProcessingBasics.ipynb

```diff
@@ -4,42 +4,436 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Lab 02: Basic image image processing\n",
+    "# Lab 02: Basic image processing\n",
     "\n",
-    "The purpose of this lab is to practice doing math on images in space and/or time. We will also learn how to summarize information you extract from an image into variables that you can do statistics on or export for statistics in another program. "
+    "The purpose of this lab is to introduce programs from FSL and AFNI for doing math on images in space and/or time. We will also learn how to summarize information you extract from an image into variables that you can then plot to help understand what just happened to the image."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "## Preparation \n",
-    "* Practice with variables math in jupyter notebook\n",
-    "* Download imaging data"
+    "* import python tools for plotting\n",
+    "* learn how to make common plots of timeseries data and images that will be helpful later"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn as sns\n",
+    "import numpy as np\n",
+    "from nilearn import plotting\n",
+    "from nilearn import image"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Plot a timeseries"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# the np below refers to the numpy toolbox we imported above and renamed to np\n",
+    "np.random.seed(seed=1)\n",
```

Comment:
This will set it so everyone will get an identical result when they use np.random (as in everyone in the class will get the same result when they run the command once, not that np.random will only return one result if you keep running np.random multiple times)